src/com/foamdev/demos/TwoWayDataBinding/Controller.js

```diff
@@ -0,0 +1,84 @@
+/**

```

Comment:
We already have com.google.foam.demos.sevenguis.TempConv. What is this version demonstrating differently? Just how do do it as a Controller or with a more customized view?